### DISCUSSION TITLE
-b is not working

### DISCUSSION THREAD

**Discussion by Hcerit07** [Chia-Network/bladebit#220] [Q&A] [answered]
./bladebit -t 14 -c < > -f < > diskplot -a --cache 99G -t1 /mnt/temp/ /mnt/pw/ -b 64

./bladebit -t 14 -c < > -f < > -b 64 diskplot -a --cache 99G -t1 /mnt/temp/ /mnt/pw/ 

Fatal Error:  
Unexpected argument '-b'

both of settings i recieve this alert

**Comment by harold-b** [comment, marked as answer]
`./bladebit -t 14 -c < > -f < > -b 64 diskplot -a --cache 99G -t1 /mnt/temp/ /mnt/pw/`
`-b` is an option that belongs to the `diskplot` command, so it needs to go sometime after `diskplot` but not as the last argument, which is the output directory. So this should be:

`./bladebit -t 14 -c < > -f < >  diskplot -b 64 -a --cache 99G -t1 /mnt/temp/ /mnt/pw/`

**Reply by Hcerit07** [reply]
thanks
